tools/roxvet/common/file_filters.go

```diff
@@ -15,14 +15,15 @@ func IsGeneratedFile(f *ast.File) bool {
 	if len(f.Comments) == 0 {
 		return false
 	}
-	firstComment := f.Comments[0].List[0]
-
-	if firstComment.Pos() > f.Package {
-		// comment is not at beginning of file, before the package keyword
-		return false
+	for _, comments := range f.Comments {
+		for _, comment := range comments.List {
+			if generatedCodeRegex.MatchString(comment.Text) {
```

Comment:
nit: also let's make sure we only take into account comments which are _before_ `f.Package` and short-circuit here if they are not.